Dockerfile

```diff
@@ -1,36 +1,37 @@
-# Basic image
-FROM openjdk:11
+# Base image for building lpvs lib
+FROM openjdk:11 AS builder
 
-# Install dependencies and remove tmp files
+# Install dependencies
 RUN apt-get update && \
-apt-get upgrade -y && \
-apt-get install -y python3-pip maven && \
-apt-get clean && \
-rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    apt-get upgrade -y && \
+    apt-get install -y maven
 
-# Setup env variables
-ENV PRJ_DIR="LPVS"
+# Copy source code into container
+WORKDIR /root/
+COPY . .
 
-# Create project dir
-RUN mkdir $PRJ_DIR
+# Build LPVS-open-source application
+RUN mvn clean install
 
-# Set workdir
-WORKDIR /$PRJ_DIR
+# Base image for running lpvs container
+FROM openjdk:11
 
-# Copy source code into container
-COPY . .
+# Install dependencies and remove tmp files
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y python3-pip && \
+    apt-get clean
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install SCANOSS
 RUN pip3 install scanoss
 
-# Build LPVS-open-source application 
-RUN mvn clean install
-
 # Allow to listen port 7896
 EXPOSE 7896
 
 # Set workdir for running jar
-WORKDIR /$PRJ_DIR/target
+WORKDIR /root/
+COPY --from=builder /root/target/lpvs-*.jar ./lpvs.jar
 
 # Run application in container
-CMD ["java", "-jar", "lpvs-1.0.2.jar"]
```

Comment:
Cancel changes

Reply:
It's the correct change. It is inefficient to increase the number every time when we upgrade the version. 
We can know the lpvs version from logs.
```
2023-05-19 13:02:05.206  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : Starting LicensePreValidationSystem v1.0.2 using Java 11.0.16 on 84cc23054a0c with PID 1 (/root/lpvs.jar started by root in /root)
```